cloud-infrastructure/storage/block-storage/README.md

```diff
@@ -0,0 +1,71 @@
+# Block Storage
+
+High-performance block storage at any scale
```

Comment:
End of sentence missing. Add a period ('.').

Reply:
done